docs/05_testing.rst

```diff
@@ -439,99 +447,190 @@ If you are curious to see which categories are available in FogLAMP, simply type
+    "OMFHttpTimeout":    { "description": "Timeout in seconds for the HTTP operations with the OMF PI Connector Relay",
+                           "type": "integer", "value": "30", "default": "30" },
+    "StaticData":        { "description": "Static data to include in each sensor reading sent to OMF.",
+                           "type": "JSON", "value": "{\"Company\": \"Dianomic\", \"Location\": \"Palo Alto\"}", "default": "{\"Company\": \"Dianomic\", \"Location\": \"Palo Alto\"}" },
+    "duration":          { "description": "How long the sending process should run (in seconds) before stopping.",
+                           "type": "integer", "value": "60", "default": "60" },
+    "sleepInterval":     { "description": "A period of time, expressed in seconds, to wait between attempts to send readings when there are no readings to be sent.",
+                           "type": "integer", "value": "5", "default": "5" },
+    "source":            { "description": "Defines the source of the data to be sent on the stream, this may be one of either readings, statistics or audit.",
+                           "type": "string", "value": "readings", "default": "readings" },
+    "blockSize":         { "description": "The size of a block of readings to send in each transmission.",
+                           "type": "integer", "value": "5000", "default": "5000" },
+    "applyFilter":       { "description": "Whether to apply filter before processing the data",
+                           "type": "boolean", "value": "False", "default": "False" },
+    "stream_id":         { "description": "Stream ID",
+                           "type": "integer", "value": "1", "default": "1" }
   }
   $
 
-What we need to do now is to change the IP address in the URL of *SEND_PR_1* with the IP address of the Windows Server running the PI Connector Relay. Supposing the Windows IP address is *192.168.56.101*, we can use curl for this:
+You can look at the descriptions to have a taste of what you can control with this plugin. The configuration should be fine, with the exception of the *URL*, which of course should refer to the IP address of the machine and the port used by the PI Connector Relay OMF. Assuming, for example, that the port is *5460* and the IP address is *192.168.56.101*, you can set the new URL with this PUT method:
 
 .. code-block:: console
 
-  $ curl -H 'Content-Type: application/json' -X PUT -d '{ "value": "http://192.168.56.101:8118/ingress/messages" }' http://localhost:8081/foglamp/category/SEND_PR_1/URL;echo
-  {"value": "http://192.168.56.101:8118/ingress/messages", "default": "http://WIN-4M7ODKB0RH2:8118/ingress/messages", "description": "The URL of the PI Connector to send data to", "type": "string"}
+  $ curl -sH'Content-Type: application/json' -X PUT -d '{ "value": "https://192.168.56.101:5460/ingress/messages" }' http://localhost:8081/foglamp/category/SEND_PR_1/URL | jq
```

Comment:
we maybe highlight that the port 5460 (and https) is for the Connect Relay 1.2 (the previous one was using 8118 and http)